packages/filesystem/src/browser/file-dialog/file-dialog-tree-filters-renderer.tsx

```diff
@@ -38,6 +39,7 @@ export class FileDialogTreeFiltersRenderer extends ReactRenderer {
 
     constructor(
         readonly filters: FileDialogTreeFilters,
+        readonly allFilesPosition: FileDialogTreeAllFilesPosition | undefined,
```

Comment:
it's a breaking change, it would be better to introduce optional new arg of `object` type, i.e.
```
export interface FileDialogTreeFiltersOptions {
    allFilesPosition?: FileDialogTreeAllFilesPosition
}

constructor(filters, filedDialogTree, options?: FileDialogTreeFiltersOptions)
```

Reply:
Good point, I'll change that.